### DISCUSSION TITLE
Access violation error & crash when trying to start game

### DISCUSSION THREAD

**Discussion by Lokizypresse902** [ppy/osu#27139] [Q&A]
### Type

Crash to desktop

### Bug description

Osu!lazer sometimes crashes when I alt-tab out of it. I can easily replicate it by spamming alt-tab in Song-select or in the main menu. 
I haven't tested it anywhere else 

### Version

osu! 2022.319.0-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/8309942/network.log)
[performance.log](https://github.com/ppy/osu/files/8309943/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8309944/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8309945/updater.log)
[database.log](https://github.com/ppy/osu/files/8309946/database.log)

**Comment by bdach** [comment]
possibly duplicate of https://github.com/ppy/osu/issues/12480 but there is not enough to go on here, logs do not contain a crash stack trace. please check windows event viewer for errors that look related to lazer.

**Comment by Lokizypresse902** [comment]
I found two of them
First:
Application: osu!.exe
CoreCLR Version: 6.0.222.6406
.NET Version: 6.0.2
Description: The process was terminated due to an internal error in the .NET Runtime at IP 00007FF97B1CD72F (00007FF97B000000) with exit code c0000005.

Second (on German):
Name der fehlerhaften Anwendung: osu!.exe, Version: 0.0.0.0, Zeitstempel: 0x61e1d8d9
Name des fehlerhaften Moduls: coreclr.dll, Version: 6.0.222.6406, Zeitstempel: 0x61e1d09e
Ausnahmecode: 0xc0000005
Fehleroffset: 0x00000000001cd72f
ID des fehlerhaften Prozesses: 0x2c48
Startzeit der fehlerhaften Anwendung: 0x01d83be60abcead2
Pfad der fehlerhaften Anwendung: C:\Users\DELL PC\AppData\Local\osulazer\app-2022.319.0\osu!.exe
Pfad des fehlerhaften Moduls: C:\Users\DELL PC\AppData\Local\osulazer\app-2022.319.0\coreclr.dll
Berichtskennung: f5814d9a-6d73-4060-b575-e6522bcd2bd5
Vollständiger Name des fehlerhaften Pakets: 
Anwendungs-ID, die relativ zum fehlerhaften Paket ist: 

Should I send the details section as well?

**Comment by bdach** [comment]
that will be the actual issue. [a lot of people are reporting similar crashes](https://github.com/ppy/osu/search?q=c0000005&type=issues), the error is a generic "access violation" error and we can't seem to get any more useful info about it. the only thing that has helped people is graphics device driver / OS updates and only sometimes. some users that had nvidia shadowplay said that turning it off also fixes it, some other people claimed that turning off "memory integrity" checks in windows fixes it - not sure about that second one though, sounds dodgy.

i can't offer much more help than that without owning the exact same hardware, sorry. going to keep this open as a general reference issue to point people to.

**Comment by Lokizypresse902** [comment]
Alright. For me, it's not that much of a deal because I simply don't mind it that much since I can just restart it and it doesn't happen too often.
I just want to state that I _can_ start the game and play. It doesn't crash when I just _start_ the game.
Another thing: It happened in version 2022.314 as well.
And finally: None of the things that you mentioned work for my case, except the "turning off memory integrity". I haven't tried that but I'd rather not either

**Comment by peppy** [comment]
@Lokizypresse902 can you provide a dxdiag for your system?

**Comment by Lokizypresse902** [comment]
@peppy 
![image](https://user-images.githubusercontent.com/58377445/159168589-06a1c7e3-ea55-4755-981b-f9fc36f77d01.png)

**Comment by smoogipoo** [comment]
The screenshot doesn't help, but the file provided form the "all information" button at the bottom would. Regardless, check that you have the most recent GPU drivers installed.

**Comment by Lokizypresse902** [comment]
[DxDiag.txt](https://github.com/ppy/osu/files/8320014/DxDiag.txt)
the integrated gpu is really weird since the cpu is really old so i can't really find proper drivers

**Comment by smoogipoo** [comment]
You can get drivers directly from your manufacturer: https://www.dell.com/support/home/en-us/product-support/product/optiplex-3010/drivers, and/or they should appear in Windows Update also.

The latest driver seems to be from 2019, whereas you have 2016.

**Comment by Lokizypresse902** [comment]
Thanks. I installed them, but it doesn't fix the problem

**Comment by AsuToki360** [comment]
I'm having the same issue on one of my laptops. Here's my DxDiag just in case, and yes I have the drivers updated.
(This laptop used to have 8GB of RAM but for some reason it's 4)
![image](https://github.com/ppy/osu/assets/40326499/563976be-8533-4614-a73e-178a2d994414)
